### PR TITLE
corrected regex patterns

### PR DESCRIPTION
**Pull request by sebbrandt87**
for any case

### CODE REVIEW

File: src/collectors/mysql/mysql.py [hunk 1]
```diff
@@ -434,7 +434,7 @@ def collect(self):
 
         for host in self.config['hosts']:
             matches = re.search(
-                '^([^:]*):([^@]*)@([^:]*):?([^/]*)/([^/]*)/?(.*)', host)
+               '^([^:]*):([^@]*)@([^:]*):?([^/]*)/([^/\]]*)/?(.*)]?$', host)
```
Comment (jaingaurav) [inline src/collectors/mysql/mysql.py]: Seems this is 3 column indented instead of 4

**Comment by jaingaurav** [comment]
Note sure I fully understand what this diff fixes. Could you please provide a test case to help us understand?

**Comment by sebbrandt87** [comment]
Yes for sure, will do so later.

Gaurav Jain <notifications@github.com> schrieb am Sa., 5. Sep. 2015 22:37:

**Comment by sebbrandt87** [comment]
sry was totally my mistake, missconfigured diamond.conf

**Pull request closed without merging**